### PR TITLE
#37 updated pytest version in requirements.txt

### PR DESCRIPTION
**Pull request by aryanbattula**
- I updated he requirements to have pytest version 7.2.1.

### CODE REVIEW

**Comment by thomasmholder** [comment]
Ok I lied, we don't need to update the version number, go ahead and re-request review and I'll approve it.

File: requirements.txt [hunk 1]
```diff
@@ -5,5 +5,5 @@ packaging==21.3
 pluggy==1.0.0
 py==1.11.0
```
Comment (DarylDohner) [inline requirements.txt:6]: need to remove py==1.11.0

**Review by DarylDohner — CHANGES_REQUESTED** [review]

**Comment by DarylDohner** [comment]
@aryanbattula you good?

**Review by DarylDohner — CHANGES_REQUESTED** [review]
Looks like you still gotta do 2 things:

1.  add your name to contributors.md
2. you got a merge conflict you gotta resolve, lmk if you need help with this

**Comment by thomasmholder** [comment]
Looks good to me, @DarylDohner anything else?

**Review by thomasmholder — APPROVED** [review]

**Review by DarylDohner — APPROVED** [review]

**Pull request merged**